source/agora/utils/Test.d

```diff
@@ -385,6 +385,42 @@ public struct TxBuilder
         return this;
     }
 
+    /***************************************************************************
+
+        Attaches to a range of tuples.
+        The tuple should have an `Output` and a `Hash` as its elements.
```

Comment:
```suggestion
        Attaches to a range of tuples.

        The tuple should have an `Output` and a `Hash` as its first and
        second elements, respectively.
```

The newline matters, as it's the separator between [summary and description](https://dlang.org/spec/ddoc.html#sections). And specifying the order (and how it's accessed) matters to the reader.